docs/src/content/docs/installation.mdx

```diff
@@ -59,7 +59,7 @@ You can follow the installation instructions in the
       environment variable.
 
       ```sh title=".zshrc"
-      export PATH = "$PATH:/home/user/.pesde/bin"
+      export PATH="$PATH:/$HOME/.pesde/bin"
```

Comment:
Since `$HOME` is equal to something like `/home/user` this will result in a double slash.
```suggestion
      export PATH="$PATH:$HOME/.pesde/bin"
```